### PR TITLE
Roll Number input is now case insensitive

### PR DESCRIPTION
**Pull request by kry0sc0pic**
Fixing #8

### CODE REVIEW

**Comment by sloth-ontabasco** [comment]
I think a better solution would be to add `re.IGNORECASE` to the `re.fullmatch()` function in line no 55 instead of mutating the input. 
Doesn't really matter tho, just a personal preference.

**Comment by kry0sc0pic** [comment]
I noticed the regex only checking for lower case after i made the PR, mb.

File: .vscode/settings.json
Comment (IndraReddy5) [inline .vscode/settings.json:1]: Is it possible for you to remove this file? using gitignore

**Comment by IndraReddy5** [comment]
Just add .vscode/* to gitignore

**Comment by sinsniwal** [comment]
> I think a better solution would be to add `re.IGNORECASE` to the `re.fullmatch()` function in line no 55 instead of mutating the input. Doesn't really matter tho, just a personal preference.

Was thinking of converting that roll number into lowercase so that our database also has them in lowercase

**Comment by sloth-ontabasco** [comment]
> > I think a better solution would be to add `re.IGNORECASE` to the `re.fullmatch()` function in line no 55 instead of mutating the input. Doesn't really matter tho, just a personal preference.
> 
> Was thinking of converting that roll number into lowercase so that our database also has them in lowercase

Ah I see! Data sanitizing could be done on the web backend also, but it's just personal preference like I mentioned.

**Comment by sinsniwal** [comment]
> > > I think a better solution would be to add `re.IGNORECASE` to the `re.fullmatch()` function in line no 55 instead of mutating the input. Doesn't really matter tho, just a personal preference.
> > 
> > 
> > Was thinking of converting that roll number into lowercase so that our database also has them in lowercase
> 
> Ah I see! Data sanitizing could be done on the web backend also, but it's just personal preference like I mentioned.

okay, for now, let's leave that to the website backend.

**Review by sinsniwal — APPROVED** [review]

**Pull request merged**